deeppavlov/configs/odqa/ranker_prod.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dataset_iterator": {
+    "name": "sqlite_iterator",
+    "data_path": "odqa",
+    "batch_size": 10000,
+    "data_url": "http://lnsigo.mipt.ru/export/datasets/wikipedia/wiki_full.db"
+  },
+  "model": {
```

Comment:
All configs should be in the new format like  
"chainer": {
    "in": ["x"],
    "pipe": [ ...

Reply:
It seems that due to old config format you model can't be deployed with REST API and your model can't be added to demo UI.

Reply:
Current Chainer class is not applicable to my model. I can try to refactor the class, but then we won't deploy in time anyway. So what looks faster to you - refactoring Chainer or changing the deployment code so it would work with old configs?

Reply:
Without unified config structure and with partially broken functions the model won't be added to release. It will stay in the current branch. I will make a patch to deployment code to run your model on demo stand.

Reply:
Ok then, so I will refactor the Chainer class.